CONTRIBUTING.md

```diff
@@ -0,0 +1,92 @@
+Creating a New Example Implementation
+============
+
+Getting Started
+---
+1. [Fork this repo](https://github.com/ericelliott/todotasks/fork)
+
+2. clone your fork and copy the `base` files
+    ```
+    # clone!
+    git clone https://github.com/<your-fork>/todotasks.git
+    # make a copy of the `base` implementation:
+    cp -r todotaks/implementations/base todotasks/implementations/<your-build-process-name>
+    # install dependencies
+    cd todotasks/implementations/<your-build-process-name>
+    npm install
+    ```
+
+3. Implement all the tasks listed below in your selected build process and link them to npm scripts of the same name  in `package.json`.
+
+4. Update the `README.md` with detailed information about your task runner. See below for details.
+
+5. Verify that your example covers all the below tasks by running this helper script:
```

Comment:
Maybe we should land the implementation for https://github.com/ericelliott/todotasks/issues/10 before we merge this.